### DISCUSSION TITLE
Style does not work

### DISCUSSION THREAD

**Discussion by fport** [module-federation/module-federation-examples#2995] [Q&A]
I set up 2 nextjs projects called Composer and Fragment. I use tailwind in both. I use inline styling in Fragment. When I call remote fragment on Composer, the style does not work. How can I solve it.

### Composer 
<img width="358" alt="Screenshot 2023-04-23 at 16 44 59" src="https://user-images.githubusercontent.com/56169582/233843390-a733b935-b05d-420a-8e6c-8a80dde9ef97.png">

### Fragment
<img width="207" alt="Screenshot 2023-04-23 at 16 45 09" src="https://user-images.githubusercontent.com/56169582/233843397-a84d6169-6ac2-4535-89d6-c6065cbe2c38.png">

**Comment by fport** [comment]
Hey @belinef 🫡  I've seen improvements in this regard, but do you have the possibility to make an example that explains it in a simple way and share it? Or do you have the availability to explain? Thank you very much

**Comment by ScriptedAlchemy** [comment]
https://github.com/module-federation/core/issues/3153
